### DISCUSSION TITLE
Auto fill create time [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5546] [Discord Threads]
> 🙋‍♂️ **saurav_mitra**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### There are system columns `CreatedAt` `UpdatedAt` . You can enable them from `Fields` menu, `Show System Fields` option  
#  
> 🙋‍♂️ **saurav_mitra**  
#### Enabling system fields adds a bunch of columns prefixed with `nc`, but none of them seem to have any dates on them  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### whats your scenario? connecting to existing DB? or created a new table on NocoDB UI?  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
####   
![image0](https://cdn.discordapp.com/attachments/1087329525479780433/1087339223901810789/Screenshot_2023-03-20_at_5.07.16_PM.png)  
![image1](https://cdn.discordapp.com/attachments/1087329525479780433/1087339224199614474/Screenshot_2023-03-20_at_5.07.25_PM.png)  
![image2](https://cdn.discordapp.com/attachments/1087329525479780433/1087339224518361248/Screenshot_2023-03-20_at_5.07.34_PM.png)  
#  
> 🙋‍♂️ **saurav_mitra**  
#### I connected to my postgres db by adding it as a base, then imported a bunch of tables from Airtable  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### ok. you can add a column with default value set to `now()`  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
####   
![image0](https://cdn.discordapp.com/attachments/1087329525479780433/1087341292889055292/Screenshot_2023-03-20_at_5.15.36_PM.png)  
#  
> 🙋‍♂️ **saurav_mitra**  
#### awesome, thanks 🙂  
#  
> 🙋‍♂️ **saurav_mitra**  
#### for the `UpdatedAt`, I presume there isn't a way to get a column with the date for when another column was modified? (Like the "Last modified time")  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Specific column modified time, not possible.    
#### You can get `UpdatedAt` at record level  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### `UpdatedAt` at record level   
####    
#### - create a date time column   
#### - double click on showAll/hideAll (additional developer specific options hidden under easter egg)   
#### - configure as below [note : AU -> Auto Update]  
![image0](https://cdn.discordapp.com/attachments/1087329525479780433/1087346918105022624/Screenshot_2023-03-20_at_5.36.59_PM.png)  
#  
> 🙋‍♂️ **saurav_mitra**  
#### Great, thank you for the help! Looking forward to specific columns at some point in the future, but this is enough for now 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087329525479780433)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
